Sources/Apodini/Properties/ObservedObject.swift

```diff
@@ -0,0 +1,101 @@
+import Foundation
+
+/// Property wrapper that subscribes to an `ObservableObject` and evaluates a `Handler` or `Job` on changes.
+///
+/// Refer to the documentation of
+/// [ObservedObject](https://github.com/Apodini/Apodini/blob/develop/Documentation/Communicational%20Patterns/2.%20Tooling/2.4.%20ObservedObject.md)
```

Comment:
Same as for the other doc-reference. Please add more information to the code at least if you still want to include the link.

Reply:
Changed the documentation